docs/howto.rst

```diff
@@ -44,6 +44,13 @@ You can for instance apply a cone search as shown `here
 You can also select observations based on other informations available using the
 `~gammapy.data.ObservationTable.select_observations` method.
 
+Make a on-axis equivalent livetime map
++++++++++++++++++++
+
+To compute an effective on-axis exposure in hours, see `here
```

Comment:
Maybe be a bit more explicit here with a similar argument than in the notebook, e.g. IACT detection efficiency varies in the FoV.  To have an estimate of the effective exposure, with respect to the on-axis one, one can build an on-axis equivalent lifetime map.